From dd2726f62c85457ae5dd5d793975bd5bb90a91b3 Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Wed, 15 Mar 2023 08:51:38 +0200
Subject: [PATCH] [SW-136144] common: include only device specific makefiles

Any Environmant variable initialization or overidding any
existing env variables in device specific makefile will
effect all the platforms. This patch adds support to
include only the device specific make files for compilation.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: I3f04fb8ce65834e25919b21ae358bc67e3cabc37
---

diff --git a/Makefile b/Makefile
index c2a9d8f..504ee0a 100644
--- a/Makefile
+++ b/Makefile
@@ -112,5 +112,8 @@
 	HL_PASSWD = $(HL_LAB_PASSWORD)
 endif
 
-include scripts/make/*/Makefile
+include scripts/make/common/Makefile
+include scripts/make/hl-common/Makefile
+include scripts/make/$(DEV_PREFIX)*/Makefile
+
 endif	# skip-makefile
